src/algebra/char_p.lean

```diff
@@ -163,6 +163,14 @@ begin
   rw fact at *, linarith,
 end
 
+lemma char_p.char_field_eq_char_field {K L : Type*} [field K] [field L] (f : K →+* L) (p : ℕ) :
```

Comment:
I don't like this name. How about `ring_hom.char_p_iff_char_p`?

Reply:
Good point. Fixed.